pkg/api/v2/reporter/reporter.go

```diff
@@ -137,6 +137,16 @@ func (e ResourceReports) Find(kind string, ref *core.ResourceRef) (resources.Inp
 	return nil, Report{}
 }
 
+func (e ResourceReports) FilterByKind(kind string) map[resources.InputResource]Report {
```

Comment:
Isn't map[resources.InputResource]Report the same type as ResourceReports?

Reply:
Ah good catch!